.infra/Pulumi.adhoc.yaml

```diff
@@ -44,6 +44,8 @@ config:
   api:env:
     accessSecret: topsecret
     analyticsUrl: http://localhost:3000
+    appleAppAppleId: 6740634400
```

Comment:
naming reminder me of https://www.youtube.com/watch?v=NfuiB52K7X8

Reply:
Internally it's just `appAppleId` and `bundleId`, but I wanted to group them together, so added `apple` and `appleApp` prefix to it 😬